clients/python-apiserver-client/python_apiserver_client/params/environmentvariables.py

```diff
@@ -0,0 +1,102 @@
+import enum
+
+
+class EnvarSource(enum.Enum):
+    CONFIGMAP = 0
+    SECRET = 1
+    RESOURCEFIELD = 2
+    FIELD = 3
+
+
+class EnvVarFrom:
```

Comment:
Is there any sort of "lightweight" Python library for K8s that defines stuff like this?

Reply:
Nop. The whole point here was to simplify this for novice user